src/command/command.js

```diff
@@ -121,28 +121,42 @@ yargs
         }
     })
     .command(
-        'purge [flotiqApiKey] [options]',
+        'purge [flotiqApiKey] [withInternal] [force]',
```

Comment:
[withInternal] [force] means, that there are a positionals;
i think we should leave `[withInternal] [force]` from command name - we have flags mentioned in help
```suggestion
        'purge [flotiqApiKey]',
```